src/main/java/Events.java

```diff
@@ -0,0 +1,46 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.text.DateFormat;
+
+/**
+ * returns the toString of Events
+ */
+public class Events extends Task {
+    protected String at;
```

Comment:
Maybe you can consider using `private` instead of `protected` in favour of cleaner code as it tends to violate OCP.